docs/middleware/android-sdk.md

```diff
@@ -1,190 +1,126 @@
-# Sourcing Entity - Android SDK
-FinBox Lending SDK is a one stop solution to seamlessly onboard users to a loan application journey.
+# FinBox Lending: Android
 
-## Modules
-The SDK is broken down into multiple modules which can be customizable as per requirements. Complete SDK has following modules:
-1. Onboarding
-2. Loan dashboard
+FinBox Lending SDK is a drop-in module which can add a digital lending journey to any mobile application
+The SDK has following modules:
+
+1. OnBoarding
+2. Loan Dashboard
 3. KYC
-4. Loan offer and agreement
+4. Loan offer and Agreement
+
+## OnBoarding module
```

Comment:
Instead of ##, use ### so that the modules don't appear in the sidebar. Also, you can put these ### under a new section ##Modules